.golangci.yml

```diff
@@ -1,26 +1,33 @@
+run:
+  deadline: 1m
+
 linters:
-  disable-all: true
-  enable:
-    - errcheck
-    - golint
-    - ineffassign
-    - unconvert
-    - misspell
+  enable-all: true
+  disable:
+    - gosimple
+    - gocyclo
+    - gochecknoinits
     - govet
-    - unused
-    - deadcode
+    - golint
```

Comment:
this should be enabled

Reply:
it currently has about 20+ errors, can spend time next week enabling it

Reply:
will enable this after @colin-axner 's testing refactors have been merged